src/blob.rs

```diff
@@ -34,6 +34,11 @@ impl<'repo> Blob<'repo> {
         }
     }
 
+    /// Get the size of this blob.
+    pub fn size(&self) -> usize {
```

Comment:
Is there a reason to choose `usize` instead of `git_off_t`?  I think (although unlikely) this will silently truncate on 32-bit systems.

Reply:
Indeed, it should be a 64-bit integer type, but I'm quite confused which type should be used here. git2 doesn't seem to return any value of a "raw" type (like `git_off_t`) so I picked `u64`, as a size should be an unsigned integer.